.circleci/config.yml

```diff
@@ -195,7 +200,7 @@ workflows:
             branches:
               only:
                 - "1.8"
-                - master
+                - "2.0"
```

Comment:
(AFAIK) we don't run any perf tests off the 2.0 branch now, is this prep for future work? Should we have 1.9 and/or master-1.x in here as well?

Reply:
I have perftest work nearing completion in 2.0. 1.9 and master-1.x are not in the mix yet, though will be down the road. I changed master to 2.0 because the completion of perftest work there is near, and master is not applicable yet.